app/assets/stylesheets/_component.scss

```diff
@@ -1,6 +1,8 @@
 // bulma customize variables
 
-$family-sanserif: 'Futura', -apple-system, BlinkMacSystemFont, "Helvetica Neue", YuGothic, "ヒラギノ角ゴ ProN W3", Hiragino Kaku Gothic ProN, Arial, "メイリオ", Meiryo, sans-serif;
+$family-sanserif: "Futura", -apple-system, BlinkMacSystemFont, "Helvetica Neue",
```

Comment:
Is the new line necessary or just auto fix by the lint?

Reply:
Auto fix by the lint.
.stylelintrc and my vscode's formatter(prettier) fixed automatically.